assets/scss/helpers/_aspect-ratio.scss

```diff
@@ -0,0 +1,20 @@
+@mixin aspect-ratio($width, $height) {
```

Comment:
This seems like a new bit of functionality - worth splitting out into a separate commit so it can have its own explanation?

Reply:
Done 👍